src/applications/gi/components/search/OnlineClassesFilter.jsx

```diff
@@ -13,25 +12,17 @@ function OnlineClassesFilter({
   handleInputFocus,
   gibctBenefitFilterEnhancement,
 }) {
-  // prod flag for story BAH-13929
-  const radioButtonsLabelText = environment.isProduction()
-    ? 'How do you want to take classes?'
-    : 'Will you be taking any classes in person?';
-  // prod flag for story BAH-13929
-  const options = environment.isProduction()
-    ? [
-        { value: 'yes', label: 'Online only' },
-        { value: 'no', label: 'In person only' },
-        { value: 'both', label: 'In person and online' },
-      ]
-    : [{ value: 'no', label: 'Yes' }, { value: 'yes', label: 'No' }];
+  const options = [
+    { value: 'no', label: 'Yes' },
+    { value: 'yes', label: 'No' },
+  ];
   if (gibctBenefitFilterEnhancement) {
     return (
       <div className="filter-additional-info vads-u-margin-bottom--4">
         <AdditionalInfo triggerText="Your housing allowance">
           <RadioButtons
             label={renderLearnMoreLabel({
-              text: radioButtonsLabelText,
+              text: 'Will you be taking any classes in person?',
```

Comment:
Same as above. Keep the constant, since it is used twice.

Reply:
Constants are back in.